subprojects/docs/src/samples/groovy/incrementalCompilation/groovyIncrementalCompilation.conf

```diff
@@ -0,0 +1,9 @@
+commands: [{
+    execution-subdirectory: groovy
+    executable: gradle
+    args: compileGroovy -Dorg.gradle.groovy.compilation.avoidance=true
```

Comment:
Why are we using a system property to enable this vs a "feature preview"?

Reply:
@wolfs I remember I asked you the same question, and I forgot your answer...

Reply:
Yeah, I did as well ;)
I talked to @big-guy, and it seems like a good idea to use a feature preview instead: https://github.com/gradle/gradle/issues/9870